src/nft/pages/collection/index.tsx

```diff
@@ -45,38 +45,48 @@ const Collection = () => {
 
   return (
     <Column width="full">
-      <Box width="full" height="160">
-        <Box
-          as="img"
-          maxHeight="full"
-          width="full"
-          src={collectionStats?.bannerImageUrl}
-          className={`${styles.bannerImage}`}
-        />
-      </Box>
-
-      {collectionStats && (
-        <Row paddingLeft="32" paddingRight="32">
-          <CollectionStats stats={collectionStats} isMobile={isMobile} />
-        </Row>
-      )}
-      <Row alignItems="flex-start" position="relative" paddingX="48">
-        <Box position="sticky" top="72" width="0">
-          {isFiltersExpanded && (
-            <Filters traitsByAmount={collectionStats?.numTraitsByAmount ?? []} traits={collectionStats?.traits ?? []} />
+      {collectionStats && contractAddress ? (
+        <>
+          {' '}
+          <Box width="full" height="160">
+            <Box
+              as="img"
+              maxHeight="full"
+              width="full"
+              src={collectionStats?.bannerImageUrl}
+              className={`${styles.bannerImage}`}
+            />
+          </Box>
+          {collectionStats && (
+            <Row paddingLeft="32" paddingRight="32">
+              <CollectionStats stats={collectionStats} isMobile={isMobile} />
+            </Row>
           )}
-        </Box>
+          <Row alignItems="flex-start" position="relative" paddingX="48">
+            <Box position="sticky" top="72" width="0">
+              {isFiltersExpanded && (
+                <Filters
+                  traitsByAmount={collectionStats?.numTraitsByAmount ?? []}
+                  traits={collectionStats?.traits ?? []}
+                />
+              )}
+            </Box>
 
-        {/* @ts-ignore: https://github.com/microsoft/TypeScript/issues/34933 */}
-        <AnimatedBox
-          style={{
-            transform: gridX.interpolate((x) => `translate(${x as number}px)`),
-            width: gridWidthOffset.interpolate((x) => `calc(100% - ${x as number}px)`),
-          }}
-        >
-          {contractAddress && <CollectionNfts contractAddress={contractAddress} />}
-        </AnimatedBox>
-      </Row>
+            {/* @ts-ignore: https://github.com/microsoft/TypeScript/issues/34933 */}
+            <AnimatedBox
+              style={{
+                transform: gridX.interpolate((x) => `translate(${x as number}px)`),
+                width: gridWidthOffset.interpolate((x) => `calc(100% - ${x as number}px)`),
+              }}
+            >
+              <CollectionNfts collectionStats={collectionStats} contractAddress={contractAddress} />
+            </AnimatedBox>
+          </Row>
+        </>
+      ) : (
+        // TODO: Put no collection asset page here
+        !isLoading && <div className={styles.noCollectionAssets}>No collection assets exist at this address</div>
```

Comment:
I moved this logic here and it will be the only thing that renders now when no collections exists (besides the navbar).  The previous logic wasn't working prior to my changes and also included some filter stuff that isn't relevant.